tests/fixtures/course.json

```diff
@@ -1391,5 +1391,65 @@
 			}
 		],
 		"status_code": 200
+	},
+	"add_grading_standards": {
+		"method": "POST",
+		"endpoint": "courses/1/grading_standards",
+		"data": {
+			"title": "Grading Standard 1",
+			"id": 1,
+			"context_id": 1,
+			"context_type": "Course",
+			"grading_scheme": [
+			  {"name": "A", "value": 0.9},
+			  {"name": "B", "value": 0.8},
+			  {"name": "C", "value": 0.7}
+			]
+		  },
+		"status_code": 200
+	},
+	"get_grading_standards": {
+		"method": "GET",
+		"endpoint": "courses/1/grading_standards",
+		"data": [
+			{
+			"title": "Grading Standard 1",
+			"id": 2,
```

Comment:
It's perfectly valid for this to have an ID of 2, however changing it from 1 to 2 was unnecessary. My original comment on ID numbers was to change the ID of "Grading Standard 2" so it didn't have the same ID as "Grading Standard 1" (they were previously both 1)